src/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -98,6 +98,7 @@ UnlockFile
 UnmapViewOfFile
 VirtualAlloc
 VirtualFree
+GlobalMemoryStatusEx
```

Comment:
i missed when i made the change https://github.com/dotnet/coreclr/commit/d973d04a9295b06723ab1e5d4fcd81525417800e